### PR TITLE
adding example for custom valueOptions attributes to docs

### PR DESCRIPTION
**Pull request by matwright**
adding example to doc for #446

### CODE REVIEW

**Comment by gianarb** [comment]
Hi! @matwright can you replace codeblock with this syntax
```
```php
``` 
Thanks!

File: docs/form-element.md [hunk 1]
```diff
@@ -1,158 +1,195 @@
...
+    {
+        $this->objectManager = $objectManager;
+    }
+    
+    public function getObjectManager()
+    {
+        return $this->objectManager;
+    }    
+}
+```php
```
Comment (Bilge) [inline docs/form-element.md]: You only open a code block with a language tag. The closing tag is unchanged (just ```` ``` ````).

**Comment by matwright** [comment]
Hi @gianarb I've updated the codeblocks to php (and html for the last one)

File: docs/form-element.md (hunk 1 above)
Comment (matwright) [inline docs/form-element.md]: @Bilge yes I corrected and recommitted straight after, thanks for the headsup

**Comment by gianarb** [comment]
:+1:

**Comment by gianarb** [comment]
Thanks

**Comment by matwright** [comment]
you're welcome

**Pull request merged**